common/scripts/x86_64/Ubuntu_14.04/_helpers.sh

```diff
@@ -228,9 +228,17 @@ __registry_login() {
     mkdir -p ~/.aws
     mv -v $credentials_file ~/.aws
     if [ "$NO_VERIFY_SSL" == true ]; then
-      local docker_login_cmd=$(aws ecr --no-verify-ssl --region us-east-1 get-login)
+      if [[ "$INSTALLED_DOCKER_VERSION" != *"1.13"* ]]; then
+        local docker_login_cmd=$(aws ecr --no-include-email --no-verify-ssl --region us-east-1 get-login)
+      else
+        local docker_login_cmd=$(aws ecr --no-verify-ssl --region us-east-1 get-login)
+      fi 
     else
-      local docker_login_cmd=$(aws ecr --region us-east-1 get-login)
+      if [[ "$INSTALLED_DOCKER_VERSION" != *"1.13"* ]]; then
```

Comment:
this just needs to be there once. construct docker_login_cmd in a variable based on flags and then execute it in the end